src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -709,7 +709,7 @@ private void SetModelEventOnAnnotation()
         /// </summary>
         /// <param name="modelGuid">The model unique identifier.</param>
         /// <returns></returns>
-        private ModelBase annotationModel_GetModelBase(Guid modelGuid)
+        public ModelBase annotationModel_GetModelBase(Guid modelGuid)
```

Comment:
This sets off alarm bells for me. Why not just handle this event registration in `WorkspaceModel.AddAnnotation`?